marketplacetests/marketplace/app.py

```diff
@@ -66,7 +66,9 @@ def offline_message_text(self):
 
     def wait_for_notification_message(self, message):
         element = self.marionette.find_element(*self._notification_locator)
-        Wait(self.marionette).until(lambda m: element.text == message)
+        Wait(self.marionette).until(lambda m: element.text == message,
+                                    message='Notification not found. Expected: %s | Found: %s'
+                                            % (message, element.text))
```

Comment:
`element.text` will change within the lambda, however your failure message is constructed only once.